### DISCUSSION TITLE
[Feature] Change "standard" face for a person

### DISCUSSION THREAD

**Discussion by Daniel-PT** [immich-app/immich#8558] [Feature Request]
### The feature

I can see that everytime i merge a person it takes the "latest" face on the person.
I could be cool to have the option to "set standard face" 

Meny thanks!

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by mmomjian** [comment]
Try reversing the order when you Merge.

**Comment by bo0tzz** [comment]
When viewing a person page you can use the menu in the top right to select a different thumbnail to use.

**Reply by Daniel-PT** [reply]
Ahh perfect! Sorry could not find it in the FAQ. Meny thanks!
